src/IECoreDelight/Renderer.cpp

```diff
@@ -371,7 +368,7 @@ class DelightOutput : public IECore::RefCounted
 				scalarFormat = "float";
 			}
 
-			layerName = parameter<string>( output->parameters(), "layerName", layerName );
+			layerName = parameter<string>( output->parameters(), "layername", layerName );
```

Comment:
This is technically a breaking change, so shouldn't really be included in 1.4. The reason I had spelled it `layerName` was that we're using that convention for Arnold outputs and I was hoping it would become a standard we could apply across all the renderers. At the moment every renderer backend has different conventions for how to specify this stuff, and I'm hoping to align it all over time. Maybe we keep this one the same for now, so we can safely include it in 1.4 with the other good stuff from this PR?

Reply:
Aha! Makes sense and absolutely no problem to revert back to using `layerName`.